### PR TITLE
Update ur kinematics

### LINKED ISSUE #460: Add support for UR16e
Recently released.

`ur_support` should be extended to include meshes, config, launch files and xacro macro.

### PR DESCRIPTION
**Pull request by fmauch**
In #613 I collected a couple of reasons why I think that ur_kinematics is currently flawed. This PR addresses a couple of them.

* Adds parameters for all robot models
* Removes the `kinematics_solver_attempts` as suggested in #474
* Corrects plugin name in `kinematics.yaml` files

Since I still see issues with using this plugin due to its problems regarding robot calibration I added a README to inform users about this concern.

This closes #460

### CODE REVIEW

**Comment by fmauch** [comment]
BTW: The DH parameters are extracted from https://www.universal-robots.com/articles/ur/application-installation/dh-parameters-for-calculations-of-kinematics-and-dynamics/

File: ur_kinematics/src/ur_kin.cpp [hunk 1]
```diff
@@ -23,6 +33,16 @@ namespace ur_kinematics {
     const double d6 =  0.0922;
     #endif
 
+    //#define UR10e_PARAMS
+    #ifdef UR10e_PARAMS
+    const double d1 =  0.1807;
+    const double a2 = -0.627;
```
Comment (RobertWilbrandt) [inline ur_kinematics/src/ur_kin.cpp]:
```Suggestion
    const double a2 = -0.6127;
```
Copy-paste error

**Review by RobertWilbrandt — CHANGES_REQUESTED** [review]

**Review by RobertWilbrandt — APPROVED** [review]

**Comment by gavanderhoorn** [comment]
Was this merged with a squash-merge?

Please only use those for single-commit PRs.

**Comment by fmauch** [comment]
I would have squashed the PR anyway, so we simply skipped that step.

**Comment by gavanderhoorn** [comment]
Why? All 4 commits make sense to me to be separate commits.

Thematically they don't belong together, so better to keep them as separate commits.

**Comment by fmauch** [comment]
OK, I'll keep that in mind in future.

**Pull request merged**